lib/ds/RequestExecutor.js

```diff
@@ -90,8 +90,9 @@ RequestExecutor.prototype.execute = function executeRequest(req, callback) {
 
   request(options, function onRequestResult(err, response, body) {
     if (err) {
-      var wrapper = new Error('Unable to execute http request ' + req + ': ' + err.message);
+      var wrapper = new Error('Unable to execute http request ' + req.method + ' ' + req.uri + ': ' + err.message);
       wrapper.inner = err;
+      wrapper.stack = '';
```

Comment:
This was actually not erasing, but more setting a default stack. Since mocha threw an error of it was undefined. So I just assumed the stack was being set when you threw an error. But maybe it's being set when you create it... Anyhow, it's to fix a Mocha issue. Without this Mocha will just crash.